src/scripts/test/getColor.test.js

```diff
@@ -0,0 +1,18 @@
+const colors = require('../main.js')
+
+const getColor = (item) => {
+  if (colors.find((el) => el === item)) {
+    for (let i = 0; i < colors.length; i++) {
+      if (colors[i] === item) {
+        return colors[++i] ?? colors[0];
+      }
+    }
+  } else {
+    return colors[0];
+  }
+};
+
+
+test("get next color from array", function () {
+  expect(getColor(0)).toBe(colors[0]);
```

Comment:
```suggestion
  expect(getColor("cyan")).toBe("firebrick");
```

Reply:
Tests should be clear to understand.

Reply:
Also the value 0 is unreachable from the code using the function getColor() in your app.